Simplenote/UIView+Constraints.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+extension UIView {
+    func addFillingSubview(_ view: UIView,
+                           edgeInsets: UIEdgeInsets = .zero,
+                           useSafeArea: Bool = false) {
+
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        addSubview(view)
+        view.frame = bounds.inset(by: edgeInsets)
```

Comment:
Since the insets are set via constraints, maybe we can get rid of this one?

Reply:
Indeed 👍